apps/etl-func/avro/message.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "message",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'messages'",
+  "fields": [
+    {
+      "name": "schema_version",
+      "doc": "An integer that indicates the version of the schema.",
+      "type": "int"
+    },
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "doc": "List of operations that can be done to a message. Messages are immutable so we can only perform create operations.",
+        "name": "MessageCrudOperation",
+        "symbols": ["CREATE"]
+      }
+    },
+    {
+      "name": "id",
+      "doc": "An ULID that identifies the message inside the message collection.",
+      "type": "string"
+    },
+    {
+      "name": "sender_service_id",
+      "doc": "The id of the service that sent the message.",
+      "type": "string"
+    },
+    {
+      "name": "sender_user_id",
+      "doc": "The id of the user that sent the message.",
+      "type": "string"
+    },
+    {
+      "name": "is_pending",
+      "doc": "A boolean that indicates if the message has been processed, is_pending equals false indicates that the message has not been processed yet.",
+      "type": "boolean"
+    },
+    {
+      "name": "content_subject",
+      "doc": "The subject of the message.",
+      "type": "string"
+    },
+    {
+      "name": "content_type",
+      "doc": "Indicates the type of the message.",
+      "type": [
+        "null",
+        {
+          "type": "enum",
+          "name": "MessageContentType",
+          "symbols": [
+            "GENERIC",
+            "PAYMENT",
+            "EU_COVID_CERT",
+            "SEND",
+            "PAGOPA_RECEIPT"
+          ]
+        }
+      ],
+      "default": null
+    },
+    {
+      "name": "content_payment_data_amount",
+      "doc": "An integer that indicates the amount of the payment.",
+      "type": "int",
+      "default": 0
```

Comment:
I was considering replacing the default value with `null`, as we did for `content_payment_data_notice_number`. If a message doesn't contain payment information, I believe it's better to avoid including any payment-related data.
If you agree with me I would do the same for `content_payment_data_invalid_after_due_date`.
@lucacavallaro @acuiuli

Reply:
I agree that there should not be any payment field if it's not a payment message